qtop.py

```diff
@@ -33,7 +33,7 @@
 import tempfile
 import sys
 import logging
-from qtop_py.constants import (TMPDIR, SYSTEMCONFDIR, QTOPCONF_YAML, QTOP_LOGFILE, savepath, USERPATH, MAX_CORE_ALLOWED,
+from qtop_py.constants import (SYSTEMCONFDIR, QTOPCONF_YAML, QTOP_LOGFILE, USERPATH, MAX_CORE_ALLOWED,
```

Comment:
don't get uneasy with the PR size now, it's just some long-due cleaning of savepath taking place...